src/gpuarray/buffer.h

```diff
@@ -746,10 +746,6 @@ typedef enum _ga_usefl {
   /* If you add a new flag, don't forget to update both
      gpuarray_buffer_{cuda,opencl}.c with the implementation of your flag */
   /**
-   * The kernel is made of PTX code.
-   */
-  GA_USE_PTX =      0x1000,
```

Comment:
Do we want to keep that options in case you or someone else in the futur make new kernel in ptx? The probability is low, but if the code still work we could keep it. But don't loose time to make it work if it get broken by other change.